blockchain/beacon/Cargo.toml

```diff
@@ -8,22 +8,22 @@ edition = "2021"
 features = ["json"]
 
 [dependencies]
-ahash             = "0.6"
+ahash             = "0.8"
 anyhow            = "1.0"
-async-std         = "1.9"
+async-std         = "1.12"
 async-trait       = "0.1"
 base64            = "0.13"
 bls-signatures    = { version = "0.11", default-features = false, features = ["blst"] }
```

Comment:
Can this be upgraded to `bls-signatures-0.13`?

Reply:
No, it cannot, due to `blst` version  conflict

Reply:
Great, add that as a comment.

Reply:
It's already in the `tests/serialization_tests/Cargo.toml`